java/vector/src/main/java/org/apache/arrow/vector/TimeStampMilliVector.java

```diff
@@ -111,9 +113,7 @@ public LocalDateTime getObject(int index) {
       return null;
     } else {
       final long millis = valueBuffer.getLong(index * TYPE_WIDTH);
-      final org.joda.time.LocalDateTime localDateTime = new org.joda.time.LocalDateTime(millis,
-              org.joda.time.DateTimeZone.UTC);
-      return localDateTime;
+      return DateUtility.getLocalDateTimeFromEpochMilli(millis);
```

Comment:
This change here is a change in API. It will break all applications using this getObject API.